recipe/meta.yaml

```diff
@@ -9,7 +10,7 @@ source:
   url: https://github.com/CNES/pangeo-{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
   sha256: 1497a1b6b64a57fbecc1e9d58874ee58b651f8cf29e2a80a5042443b6b19de35
 build:
-  number: 0
+  number: {{ build }}
```

Comment:
No need for a jinja variable on something that is used in a single place.

Reply:
Oops. I didn't see your point. I'm committing this change in my branch for the next time.